Task1/aga/IsPrime/IsPrime/IsPrime.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace IsPrime
+{
+    class IsPrime
+    {
+        private static int max;
+
+        public IsPrime(int val)
```

Comment:
Why calling a predicate method results in calling a setter? This should only return a boolean and not change state of the instance.